src/DonorDashboards/Tabs/Contracts/Route.php

```diff
@@ -36,7 +36,7 @@ abstract public function args();
 	 *
 	 * @param WP_REST_Request $request
 	 *
-	 * @return WP_REST_Response
+	 * @return WP_REST_Response|array
```

Comment:
I do not recommend multiple types of return values from the same function. If we do please define `@return` explicitly on class function: https://github.com/impress-org/givewp/compare/9f301c847d7d...ff646a239e29

@JasonTheAdams need your thought on this

Reply:
It's good to record in the `@return` all of the potential return types. As far as not having more than one, I don't think it's necessary to recommend one or more as a blanket statement. Functions should, as good practice, be kept simple and have a single responsibility, but that doesn't necessarily determine how many types are returned.

Reply:
In an ideal world, there would be one consistent return type coming out of this method -- however in implementation some areas of the codebase currently return an array, and other currently return a WP_REST_Response object. In a future issue it would be great to refactor these areas to all return the same type, but for now I think the doc block should reflect how the method has been actually implemented (in some contexts returning a Response object, in other contexts returning an array).